packages/core/bootstrap/src/index.ts

```diff
@@ -135,15 +140,20 @@ const withMetrics: Middleware = async (execute, context) => async (input: Adapte
   const record = recordMetrics()
   try {
     const result = await execute({ ...input, metricsMeta }, context)
-    record(
-      result.statusCode,
-      result.data.maxAge || (result as any).maxAge
-        ? metrics.HttpRequestType.CACHE_HIT
-        : metrics.HttpRequestType.DATA_PROVIDER_HIT,
-    )
+    record({
+      statusCode: result.statusCode,
```

Comment:
Should we also record the `providerStatusCode` as 200 here?

Reply:
The normalization that happens within the record function will now default that to 200, so it will be added there